### ISSUE TITLE
Suport for Nokia LCD PCD8544

### ISSUE THREAD

**Issue by rizzo86** [Edzelf/ESP32-Radio#328]
Is it posible import library

https://github.com/adafruit/Adafruit-PCD8544-Nokia-5110-LCD-library

For LCD Nokia PCD8544.

Thanks

**Comment by Edzelf** [comment]
Yes, I guess it is possible, but the resolution is to low to display the information.

**Comment by rizzo86** [comment]
I can help make better fonts...
